### ISSUE TITLE
Improve delete buttons

### ISSUE THREAD

**Issue by jschirrmacher** [mrsimpson/collaborative-shopping#96]
It could be more intuitive to delete items from the shopping list. At least, I didn't expect the edit button at the top of the list to be the path to delete items.

**Comment by mrsimpson** [comment]
Yup. I was trying with swipe-for-deletion and display the icon always when in desktop-mode, but I didn’t get the swipe to work properly. 
Feel free to try, there is a branch for that
